### PR TITLE
Fix traefik chart version to 22.4.0

### PR DESCRIPTION
**Pull request by ktsakalozos**
Fix the issue:
```
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/4523/credentials/client.config
Release "traefik" does not exist. Installing it now.
Error: execution error at (traefik/templates/deployment.yaml:3:8): ERROR: Helm >= 3.9.0 is required
```

### CODE REVIEW

**Comment by neoaggelos** [comment]
Should we instead backport https://github.com/canonical/microk8s-community-addons/pull/125?

File: addons/traefik/disable [hunk 1]
```diff
@@ -1,16 +1,11 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 set -e
 
-source $SNAP/actions/common/utils.sh
-CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
```
Comment (neoaggelos) [inline addons/traefik/disable:4]: I think this also needs a `microk8s enable helm3` before everything else.

Comment (ktsakalozos) [reply addons/traefik/disable:4]: Done

**Review by berkayoz — APPROVED** [review]
Is there a reason to use `HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"` instead of using the `microk8s-helm3.wrapper`. Other than that LGTM

**Comment by ktsakalozos** [comment]
> Is there a reason to use `HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"` instead of using the `microk8s-helm3.wrapper`. Other than that LGTM

I cannot remember what was failing and made me move from calling the wrapper to calling the helm cli directly (https://github.com/canonical/microk8s-community-addons/pull/128/commits/2bdc93ec4c5045b18831a9396f02bacd6bf82c1c).

**Pull request merged**